.jenkins/tests.Jenkinsfile

```diff
@@ -0,0 +1,185 @@
+*  https://github.com/deislabs/mystikos/blob/main/.azure_pipelines/ci-pipeline-makefile.yml
+*  https://github.com/deislabs/mystikos/blob/main/.azure_pipelines/ci-pipeline-makefile-nightly.yml
+*/
+
+pipeline {
+    agent {
+        label 'ACC-1804-DC4'
+    }
+    options {
+        timeout(time: 600, unit: 'MINUTES')
+    }
+    parameters {
+        string(name: "BRANCH_NAME", defaultValue: "main", description: "Branch to build")
+        string(name: "SCRIPTS_ROOT", defaultValue: '${WORKSPACE}/.azure_pipelines/scripts', description: "Root directory")
+        choice(name: "MYST_NIGHTLY_TEST", choices:['0','1'], description: "Enable nightly test")
+    }
+    stages {
+        stage("Build") {
+            when {
+                /* Jobs must meet any of the situations below in order to build:
+                    1. Is started manually, or by a scheduler
+                    2. Is testing a PR to main that contains more than just documentation changes
+                    3. Has MYST_NIGHTLY_TEST set to "1"
+                */
+                anyOf {
+                    triggeredBy 'UserIdCause'
+                    triggeredBy 'TimerTrigger'
+                    expression { params.MYST_NIGHTLY_TEST == "1" }
+                    allOf {
+                        anyOf {
+                            changeRequest target: 'main'
+                            branch 'staging'
+                            branch 'trying'
+                        }
+                        not {
+                            anyOf {
+                                changeset pattern: "doc/*"
+                                changeset pattern: "*(.txt|md)\$", comparator: "REGEXP"
+                            }
+                        }
+                    }
+                }
+            }
+            stages {
+                stage("Cleanup files") {
+                    steps {
+                        sh 'sudo rm -rf /tmp/myst*'
+                        sh 'df'
+                    }
+                }
+                stage('Minimum init config') {
+                    steps {
+                        checkout([$class: 'GitSCM',
+                            branches: [[name: BRANCH_NAME]],
+                            extensions: [],
+                            userRemoteConfigs: [[url: 'https://github.com/cyandevs/mystikos']]])
+                        sh """
+                           curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+                           sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu \$(lsb_release -cs) stable"
+                           sudo apt-get update
+                           while sudo lsof /var/lib/dpkg/lock-frontend | grep dpkg; do sleep 3; done
+                           sudo apt-get install build-essential python3-setuptools python3-pip llvm-7 libmbedtls-dev docker-ce azure-cli lldb-10 -y
+                           sudo chmod 666 /var/run/docker.sock
```

Comment:
Having docker.sock r/w to world is a security risk. User should be added to the docker group instead.

Reply:
New change will add the build user to the Docker group, allowing use of docker.sock